open-weather-connector/config/variables.yaml

```diff
@@ -2,10 +2,8 @@
 Variables:
   openWeatherConnector:
     # Open Weather Data endpoint
-    weatherDataUrl: 'https://api.openweathermap.org/data/2.5'
-    
+    weatherDataUrl: ""
     # Open Weather Geo endpoint
-    weatherGeoUrl: 'https://api.openweathermap.org/geo/1.0'
-    
+    weatherGeoUrl: ""
```

Comment:
why did we remove these good defaults? :thinking:  ... AppFixture will override these values, even they are not blank